src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const hasTabId = tabs.some(el => el.id === selectedTabId);
+  let correctSelectedTabId = selectedTabId;
+
+  if (!hasTabId) {
+    correctSelectedTabId = tabs[0].id;
+  }
```

Comment:
we can just try to find the tab using the find method or give some default value if the find returns undefined